ui/src/components/ConfigurationForm/ConfigurationForm.component.tsx

```diff
@@ -119,13 +165,53 @@ export const ConfigurationForm: React.FC<ConfigurationFormProps> = ({ isEmbedded
     });
   };
 
+  const [snackbar, setSnackbar] = useState<SnackbarConfig>({
+    open: false,
+    severity: 'info',
+  });
+  const handleSnackClose = (event?: SyntheticEvent | Event, reason?: string) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+    setSnackbar({ open: false });
+  };
+
+  const handleShareConfiguration = async () => {
+    const casted = form.getValues() as StarterRequest;
+    const urlToShare = stringifyUrl(
+      { url: window.location.href, query: { ...casted } },
+      { skipNull: true, skipEmptyString: true }
+    );
+    await navigator.clipboard.writeText(urlToShare);
+    setSnackbar({ open: true, severity: 'info', message: 'Link to configuration copied to clipboard.' });
+  };
+
   return (
     <Box>
       {!isEmbedded && (
-        <Typography variant="h3" component="h3" fontWeight={300} gutterBottom>
-          Generate tapir project
-        </Typography>
+        <Stack direction="row" spacing={0.25}>
+          <Typography variant="h3" component="h3" fontWeight={300} gutterBottom>
+            Generate tapir project
+          </Typography>
+          <Tooltip title="Share configuration" arrow>
+            <IconButton color="secondary" aria-label="share configuration" onClick={handleShareConfiguration}>
```

Comment:
Nit: When I have hovered over this icon I saw this:
![Screenshot from 2022-09-09 17-02-16](https://user-images.githubusercontent.com/79414144/189383427-4625fe4b-a870-4588-800c-5e5bc97fd9ab.png)
This darker background in ellipsis looks a little strange to me. Maybe circle would be better? Or changing not background of the icon, but icon color (f.e. from grey to orange?).

Reply:
I have no idea how to fix it but it bothered me too. @kkolasinski any suggestions would be welcomed. Another thing to _fix_ is that it looks a bit misaligned IMHO the share button should be a few pixels up (again have no idea on how to do it by the book ;))

Reply:
This can easily be fixed by setting the `alignItems` prop on the `Stack` component.

`<Stack direction="row" alignItems="baseline" spacing={0.25}>`

The value `baseline` aligns the button to the line height of the heading, another option would be `center`, but that centers vertically based on the container height, which still looks like the button is misaligned.

Reply:
@kkolasinski that was the power comment as it fixed both `ellipsis shape` and `alignement` issues :)

Reply:
thanks!